examples/unary-operators/unary_operators.bal

```diff
@@ -0,0 +1,24 @@
+import ballerina/io;
+
+public function main() {
+    int a = 10;
+
+    // Negate the value of `a`.
+    io:println(-a);
+    
+    // Invert the bits of `a`.
+    io:println(~a);
+
+    float b = -10.5;
+
+    // Negate the value of `b`.
+    io:println(-b);
+    
+    // Return the value of `b`.
```

Comment:
Rephrase to include using +.

E.g., Using the `+` operator returns the value of its operand expression.